e2e/e2e-scenario.sh

```diff
@@ -67,9 +67,21 @@ EOF
 cat $SCENARIO_NAME-vmss.json
 
 # Create a test VMSS with 1 instance 
-# TODO 3: Discuss about the --image version, probably go with aks-ubuntu-1804-gen2-2021-q2:latest
-#       However, how to incorporate chaning quarters?
 log "Creating VMSS"
+
+# If a custom SIG_VERSION_ID was not provided via the scenario matrix or via the command line, default to a locked 1804Gen2 image for now
+if [ -z "$SIG_VERSION_ID" ]; then
```

Comment:
nit: can ignore if too much effort

another way of doing it could be to instantiate SIG_VERSION_ID with the id we use if input isnt provided.
this will help reduce the code following here a bit, essentially, only the else will follow.

can totally leave it as is, its fine honestly. just an observation.

Reply:
to clarify: you mean instantiate it here or in e2e-local.sh if it's not provided?